### DISCUSSION TITLE
CVSSv2

### DISCUSSION THREAD

**Discussion by hippyjm** [DefectDojo/django-DefectDojo#10396] [Ideas]
Good morning. loving defect dojo.
just a quick one. is it possible to have a cvss v2 box as so many vulnarabilities jsut have vector strings for cvssv2 and do not have any information of v3 or v3.1 as of yet. maybe they never will. It would be good to be able to view the base scores for these as well. Currently i just have the information in a severity jsutification field in the generic import template, which works fine. but thought id pop an idea in

**Comment by manuel-sommer** [comment]
Hi @hippyjm, 
there are already multiple open issues which relate to this discussion. 
https://github.com/DefectDojo/django-DefectDojo/issues/8264
https://github.com/DefectDojo/django-DefectDojo/issues/9378
I guess there should be an overhaul of CVSS Scores in DefectDojo to support all CVSS versions.
